gravann/_validation_v2.py

```diff
@@ -0,0 +1,260 @@
+import numpy as np
+import pandas as pd
+import torch
+from tqdm import tqdm
+
+from gravann.polyhedral import ACC_L as POLYHEDRAL_ACC_L, U_L as POLYHEDRAL_U_L, calculate_density
+from . import compute_c_for_model, load_polyhedral_mesh, load_mascon_data
+from ._integration import ACC_trap, U_trap_opt, compute_integration_grid
+from ._losses import contrastive_loss, normalized_L1_loss, normalized_relative_L2_loss, \
+    normalized_relative_component_loss, RMSE, relRMSE
+from ._mascon_labels import ACC_L as MASCON_ACC_L, U_L as MASCON_U_L
+from ._sample_observation_points import get_target_point_sampler
+from ._utils import fixRandomSeeds
+
+
+def validation_v2(model, encoding, sample, ground_truth, use_acc=True, N_integration=500000, **kwargs):
```

Comment:
I am little concerned about the different v2s you are adding :D In the end we have "versioned" files in our code, that is bit weird. What is the reasoning for them?